src/clj/qbits/spandex.clj

```diff
@@ -202,30 +203,66 @@
                (response-headers response)
                (.getHost response))))
 
+(defn response-ex->response
```

Comment:
You can now overwrite exception-handling and make this automatic: 

```clj
(request c {... :exception-handler (fn [x] (throw (decode-exception x)))})
```

or if you want to have a map with status-code & co and only exceptions for networking issues & co : 
```clj
(request c {... :exception-handler decoding-exception-handler})
```